assets/eip-3475/interfaces/IERC3475.sol

```diff
@@ -191,7 +196,7 @@ interface IERC3475 {
      */
     event Burn(address indexed _operator, address indexed _from, Transaction[] _transactions);
     /**
-     * @dev MUST emit when approval for a second party/operator address to manage all bonds from a classId given for an owner address is enabled or disabled (absence of an event assumes disabled).
+     * @dev MUST emit when approval for a second party/operator address to manage all bonds from a classId given for an owner address is enabled or disabled (ie putting approval true and false respectively).
```

Comment:
No worries I reverted this change. 
basically, I wanted to say that if the owner of the bonds has approval for doing operations for given classId and nonceId.  but I think the current description suits better and changing it will be counter-productive.